packages/inventory-compliance/src/SystemRulesTable.js

```diff
@@ -457,7 +457,7 @@ class SystemRulesTable extends React.Component {
             passedRows = originalRows;
         }
 
-        const severityRows = this.filterBy(severity, originalRows, SEVERITY_COLUMN);
+        const severityRows = this.filterBy(severity.join('|'), originalRows, SEVERITY_COLUMN);
```

Comment:
Multiple severity filters were not handled properly - with this, we call `.match` on `medium|high` for example, which works